ui/ui-components/components/record/SampleRecordCard.tsx

```diff
@@ -569,7 +605,10 @@ const SampleRecordCard: React.FC<SampleRecordCardProps> = ({
         <AddSampleRecordModal
           properties={properties}
           show={addingRecord}
-          onRecordCreated={saveRecord}
+          onRecordCreated={(record, destinations, groups) => {
+            setRecordId(record?.id);
+            // saveRecord(record, destinations, groups);
+          }}
```

Comment:
@edmundito @evantahler 
The destination record preview endpoint now returns [groups and destinations](https://github.com/grouparoo/grouparoo/pull/3000/files#diff-bd88eccd7449c0e3abc5ef4ad73c2c2891340b2e3f21ed7a81ee5b16dcd09368R437).
Since fetchRecord is passed from the parent this change lets it function as the source of truth for the data so we no longer want to use the data returned from this but instead have fetchRecord grab the data we want to show based on id.